src/Event/PostMappingBuildEvent.php

```diff
@@ -12,9 +12,8 @@
 namespace FOS\ElasticaBundle\Event;
 
 use FOS\ElasticaBundle\Configuration\IndexConfigInterface;
-use Symfony\Contracts\EventDispatcher\Event;
 
-final class PostMappingBuildEvent extends Event
+final class PostMappingBuildEvent extends IndexEvent
```

Comment:
Maybe we should also rename it to PostIndexMappingBuildEvent?

ping @darthf1

Reply:
I have no opinion on this. It makes sense if you extend the `IndexEvent`

Reply:
Yeah, `PostIndexMappingBuildEvent` would be better.